config/install/search_api.index.chat_appointment.yml

```diff
@@ -0,0 +1,126 @@
+    - search_api.server.living_spaces
+  module:
+    - search_api_solr
+    - user
+    - dip_chat_counseling
+    - search_api
+third_party_settings:
+  search_api_solr:
+    finalize: false
+    commit_before_finalize: false
+    commit_after_finalize: false
+    debug_finalize: false
+    highlighter:
+      maxAnalyzedChars: 51200
+      fragmenter: gap
+      usePhraseHighlighter: true
+      highlightMultiTerm: true
+      preserveMulti: false
+      regex:
+        slop: 0.5
+        pattern: blank
+        maxAnalyzedChars: 10000
+      highlight:
+        mergeContiguous: false
+        requireFieldMatch: false
+        snippets: 3
+        fragsize: 0
+    mlt:
+      mintf: 1
+      mindf: 1
+      maxdf: 0
+      maxdfpct: 0
+      minwl: 0
+      maxwl: 0
+      maxqt: 100
+      maxntp: 2000
+      boost: false
+      interestingTerms: none
+    term_modifiers:
+      slop: 3
+      fuzzy: 1
+    advanced:
+      index_prefix: ''
+      collection: ''
+      timezone: ''
+    multilingual:
+      limit_to_content_language: false
+      include_language_independent: true
+id: chat_appointment
+name: 'Chat appointment'
+description: ''
+read_only: false
+field_settings:
+  field_appointmentdate:
+    label: 'Appointment date'
+    datasource_id: 'entity:dip_chat_appointment'
+    property_path: field_appointmentdate
+    type: date
+    dependencies:
+      config:
+        - field.storage.dip_chat_appointment.field_appointmentdate
+  field_counselor:
+    label: Counselors
+    datasource_id: 'entity:dip_chat_appointment'
+    property_path: field_counselor
+    type: integer
+    dependencies:
+      config:
+        - field.storage.dip_chat_appointment.field_counselor
+  field_moderators:
+    label: Moderators
+    datasource_id: 'entity:dip_chat_appointment'
+    property_path: field_moderators
+    type: integer
+    dependencies:
+      config:
+        - field.storage.dip_chat_appointment.field_moderators
+  name:
+    label: 'Counselor Name'
+    datasource_id: 'entity:dip_chat_appointment'
+    property_path: 'field_counselor:entity:name'
+    type: solr_text_unstemmed
+    dependencies:
+      config:
+        - field.storage.dip_chat_appointment.field_counselor
+      module:
+        - user
+  space:
+    label: Space
+    datasource_id: 'entity:dip_chat_appointment'
+    property_path: space
+    type: integer
+    dependencies:
+      module:
+        - dip_chat_counseling
+datasource_settings:
+  'entity:dip_chat_appointment':
+    bundles:
+      default: true
+      selected: {  }
+    languages:
+      default: true
+      selected: {  }
+processor_settings:
+  add_url: {  }
+  aggregated_field: {  }
+  auto_aggregated_fulltext_field: {  }
+  entity_type: {  }
+  language_with_fallback: {  }
+  rendered_item: {  }
+  solr_date_range: {  }
+tracker_settings:
+  default:
+    indexing_order: fifo
+options:
+  cron_limit: 50
+  index_directly: true
+  track_changes_in_references: true
+server: living_spaces
```

Comment:
This index already added in dip_counseling/modules/dip_chat_counseling/config/install/search_api.index.chat_appointment.yml